src/com/zenuml/dsl/PsiToDslConverter.java

```diff
@@ -153,11 +110,23 @@ public void visitWhileStatement(PsiWhileStatement statement) {
         processBody(statement);
     }
 
+    private void processCondition(PsiWhileStatement statement) {
+        Observable.fromArray(statement.getChildren())
+                .skipWhile(psiElement -> !isLparenth(psiElement))
+                .skip(1) // skip `(`
+                .takeWhile(psiElement -> !isRparenth(psiElement))
+                .filter(psiElement -> psiElement instanceof PsiMethodCallExpression)
```

Comment:
This won't work for case like `while(getCount() > 0)` where `psiElement` is `PsiBinaryExpression`.

Reply:
Restored the previous test case.